sdcm/nemesis.py

```diff
@@ -5843,6 +5843,7 @@ def disrupt(self):
 class DecommissionSeedNode(Nemesis):
     disruptive = True
     topology_changes = True
+    supports_high_disk_utilization = False  # Decommissioning a node consumes disk space
```

Comment:
`# Decommissioning a node cause increase of disk space across rest of the nodes`

Reply:
fixed